app/models/query/processed_applications.rb

```diff
@@ -6,20 +6,20 @@ def initialize(user)
 
     def find
       @user.office.applications.
-        includes(:evidence_check, :payment).
-        references(:evidence_check, :payment).
+        includes(:evidence_check, :part_payment).
+        references(:evidence_check, :part_payment).
         where(where_condition).
         order(id: :asc)
     end
 
     def where_condition
-      <<WHERE
-(applications.application_outcome IS NOT NULL)
-AND
-(evidence_checks.id IS NULL OR evidence_checks.completed_at IS NOT NULL)
-AND
-(payments.id IS NULL OR payments.completed_at IS NOT NULL)
-WHERE
+      <<-WHERE.gsub(/^\s+\|/, '')
```

Comment:
Isn't this unnecessary gsub on every sql query? I can see that it fixes indentation, but I think it's not worth doing.